common/include/Utilities/FolderUtils.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+#include "FixedPointTypes.inl"
+#include "ghc/filesystem.h"
+#include "Path.h"
+
+namespace fs = ghc::filesystem;
+
+
+namespace Path
+{
+    extern bool IsRelative(const std::string& path);
+
+    /************************************************************************
+     * @brief Indicates whether dir2 is contained within dir1
+     * @param dir1 Directory that may or may not contain the second directory
+     * @param dir2 The second directory
+     * @return true/false
+     *************************************************************************/
+    extern bool IsDirectoryWithinDirectory(fs::path base, fs::path dir);
+
+    // Returns -1 if the file does not exist.
+    extern s64 GetFileSize(const fs::path& file);
```

Comment:
`extern`s are not required, as I've learned here https://github.com/PCSX2/pcsx2/pull/4353